tools/esp32/Config.mk

```diff
@@ -147,12 +147,12 @@ endif
 
 ESPTOOL_BAUD ?= 921600
 
-# DOWNLOAD -- Download binary image via esptool.py
+# FLASHCMD -- Upload a binary image via esptool.py
```

Comment:
````suggestion
# FLASHCMD -- Download a binary image via esptool.py
```
According to the Embedded Systems Dictionary, by Jack Ganssle, `download` is the defined convention for "moving data or code into a target's memory over a communications channel such as a serial port".